src/PHPSemVerChecker/Analyzer/ClassMethodAnalyzer.php

```diff
@@ -70,16 +71,22 @@ public function analyze(Stmt $contextBefore, Stmt $contextAfter)
 
 		foreach ($methodsToVerify as $method) {
 			/** @var \PhpParser\Node\Stmt\ClassMethod $methodBefore */
-			$methodBefore = $methodsBeforeKeyed[$method];
+			$methodBefore = $methodsBeforeKeyed[strtolower($method)];
 			/** @var \PhpParser\Node\Stmt\ClassMethod $methodAfter */
-			$methodAfter = $methodsAfterKeyed[$method];
+			$methodAfter = $methodsAfterKeyed[strtolower($method)];
 
 			// Leave non-strict comparison here
 			if ($methodBefore != $methodAfter) {
-				$paramsBefore = $methodBefore->params;
-				$paramsAfter = $methodAfter->params;
 
-				$signatureResult = Signature::analyze($paramsBefore, $paramsAfter);
+				// Detect method renamed case only.
+				if(
```

Comment:
Break long comparison into variables and test against those, e.g.:
```php
$isSameName = strtolower($methodBefore->name) === strtolower($methodAfter->name);
$isExactName = $methodBefore->name === $methodAfter->name;
if ($isSameName && !$isExactName) { ... }
```

Reply:
Any reason you need to check `strtolower($methodBefore->name) === strtolower($methodAfter->name)`? They're technically mapped to be since they're part of the `$methodsToVerify` array.